src/main/java/com/cleanroommc/groovyscript/compat/mods/aetherlegacy/Accessory.java

```diff
@@ -0,0 +1,88 @@
+package com.cleanroommc.groovyscript.compat.mods.aetherlegacy;
+
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.api.documentation.annotations.*;
+import com.cleanroommc.groovyscript.helper.Alias;
+import com.cleanroommc.groovyscript.helper.recipe.AbstractRecipeBuilder;
+import com.cleanroommc.groovyscript.registry.ForgeRegistryWrapper;
+import com.cleanroommc.groovyscript.registry.ReloadableRegistryManager;
+import com.gildedgames.the_aether.api.accessories.AccessoryType;
+import com.gildedgames.the_aether.api.accessories.AetherAccessory;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+import org.jetbrains.annotations.Nullable;
+
+@RegistryDescription
+public class Accessory extends ForgeRegistryWrapper<AetherAccessory> {
+
+    private static final AccessoryType[] VALUES = AccessoryType.values();
+
+    public Accessory() {
+        super(GameRegistry.findRegistry(AetherAccessory.class), Alias.generateOf("Accessory"));
+    }
+
+    @MethodDescription(type = MethodDescription.Type.ADDITION)
+    public void add(ItemStack item, String type) {
+        AccessoryType accessoryType = AccessoryType.MISC;
+        for (AccessoryType value : VALUES) {
+            if (value.getDisplayName().equalsIgnoreCase(type)) {
+                accessoryType = value;
+            }
+        }
```

Comment:
To get the enum value of a string i recommend to use `EnumHelper.valueOfNullable()`. Additionally please log an error if the input item is null or no enum value was found. See existing compat for reference.